styles.css

```diff
@@ -1,4 +1,15 @@
-body .content .midcol > .arrow {
+body .content .midcol > .arrow,
+body .content .midcol > .arrow:hover,
+body .content .midcol > .arrow:active {
   display: block !important;
   visibility: visible !important;
+  text-indent: 0!important;
+  clip: initial!important;
```

Comment:
Could you add a space between the value and `!important` this way it matches the existing properties.